### PR TITLE
chore: Increase Snapshot Precision to 0.99

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

By having a precision of 0.90 we get a lot of false positives across the entire test suite. Tests that are supposed to fail, pass and those things don't get noticed by either devs or CI. To fix that issue we increase the Snapshot precision to 0.99. 

In addition to that I fixed a couple of tests that were failing and indeed were unnoticed. The test issues introduced by one of my cleanup PRs.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by johnxnguyen — COMMENTED** [review]
Is it necessary to update the tests because the tolerance changed? Or are they unrelated fixes?

**Comment by agisilaos** [comment]
> Is it necessary to update the tests because the tolerance changed? Or are they unrelated fixes?

It's mostly because of the change of tolerance. The ones related to network connectivity I needed to update and generate new tests because during the small refactoring I broke things.

File: wire-ios/Wire-iOS Tests/ReferenceImages/NetworkStatusViewControllerSnapshotTests/testOnlineState.1.png
Comment (netbe) [inline wire-ios/Wire-iOS Tests/ReferenceImages/NetworkStatusViewControllerSnapshotTests/testOnlineState.1.png:1]: is it normal/ expected?

Comment (agisilaos) [reply wire-ios/Wire-iOS Tests/ReferenceImages/NetworkStatusViewControllerSnapshotTests/testOnlineState.1.png:1]: Yes, this is normal and expected.

**Comment by netbe** [comment]
@agisilaos is it still needed, if not close it please

**Review by netbe — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

**Pull request closed without merging**